RNDeviceInfo.podspec

```diff
@@ -13,10 +15,13 @@ Pod::Spec.new do |s|
   s.platforms     = { :ios => "9.0", :visionos => "1.0", :tvos => "10.0"}
 
   s.source       = { :git => "https://github.com/react-native-device-info/react-native-device-info.git", :tag => "v#{s.version}" }
-  s.source_files  = "ios/**/*.{h,m}"
+  s.source_files = "ios/**/*.{h,m,mm,swift}"
   s.resource_bundles = {
     'RNDeviceInfoPrivacyInfo' => ['ios/PrivacyInfo.xcprivacy'],
   }
 
+  s.frameworks = 'CoreTelephony', 'WebKit', 'LocalAuthentication', 'DeviceCheck'
   s.dependency 'React-Core'
+  
+  install_modules_dependencies(s)
```

Comment:
If we want to support RN<0.71, you should do something like this: https://github.com/software-mansion/react-native-svg/blob/a36a676d4313225bbcec44120eb38ce871106d0a/RNSVG.podspec#L23-L34. And if not, then we don't really need `fabric_enabled = ENV['RCT_NEW_ARCH_ENABLED'] == '1'` yeah?